python/moz/l10n/bin/build.py

```diff
@@ -24,7 +24,7 @@
 
 from moz.l10n.formats import Format
 from moz.l10n.model import Comment, Entry, Resource, Section
-from moz.l10n.paths.config import L10nConfigPaths
+from moz.l10n.paths import L10nConfigPaths
```

Comment:
I get the other changes, but how this (and the change in `fix.py`) work, since the class didn't move?

Reply:
They're re-exported from the __init__ module.